packages/scene-composer/src/utils/dataBindingUtils.ts

```diff
@@ -78,7 +78,7 @@ export const dataBindingValuesProvider = (
   dataInput?: IDataInput,
   dataBinding?: IValueDataBinding,
   dataBindingTemplate?: IDataBindingTemplate,
-): Record<string, any> => {
+): Record<string, unknown> => {
```

Comment:
thank you! 🙏 lets celebrate by decrementing the max warnings down one in the package.json :-)

Reply:
that will need more effort to look through the whole data input flow typings to make sure this type is correct. the return type will be something like this at least
 ```
Record<string, number | boolean | string | number[] | boolean[] | string[]>
```
but the type of `values` filed in `dataInput` is currently `any[]`, and the value types in the return type should be derived from it.